docs/guides/skins.md

```diff
@@ -1,20 +1,33 @@
 Skins
 =====
 
-The base Video.js skin is made using HTML and CSS (although we use the [Sass preprocessor](http://sass-lang.com)), and by default these styles are added to the dom for you! That means you can build a custom skin by simply taking advantage of the cascading aspect of CSS and overriding
+The base Video.js skin is made using HTML and CSS (although we use the [Sass preprocessor](http://sass-lang.com)),
+and by default these styles are added to the dom for you!
+That means you can build a custom skin by simply taking advantage of the cascading aspect of CSS and overriding
 the styles you'd like to change.
 
-If you don't want Video.js to inject the base styles for you, you can disable it by setting `window.VIDEOJS_NO_BASE_THEME = false` before Video.js is loaded. Keep in mind that without these base styles
-enabled, you'll need to manually include them.
+If you don't want Video.js to inject the base styles for you, you can disable it by setting `window.VIDEOJS_NO_BASE_THEME = true` before Video.js is loaded.
```

Comment:
I changed `window.VIDEOJS_NO_BASE_THEME = false` to `window.VIDEOJS_NO_BASE_THEME = true` because that's how it's implemented and that's what the name implies.